test/test_loops.py

```diff
@@ -17,6 +17,42 @@ def test_while_range(self):
         py_code = 'while (x < 10):\n\t5'
         self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
         self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+        
+    def test_for_range_increment_one(self):
+        js_code = 'for (let i = 0; i < 10; i += 1) {\n\t5\n}'
```

Comment:
Is i++ supported yet? 

If it its - Can we add a test?

If not - Can we add a Taiga ticket? I suspect this will be the most commonly typed for...loop so we might want to take that task on next.

Reply:
It is not supported yet, but we are planning to add it